rustls/Cargo.toml

```diff
@@ -116,6 +116,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [package.metadata.cargo_check_external_types]
 allowed_external_types = [
-    "rustls_pki_types",
-    "rustls_pki_types::*",
+  # ---
```

Comment:
Needed a comment to keep taplo from compressing these, I can try to come up with a nicer comment here.